docs/06-contributors/workflows.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Development Workflows
+id: workflows
+keywords: [Wing contributors, contributors, workflows]
+---
+
+This topic includes a description of common development workflows for thw Wing project.
+
+## Environment Setup
+
+Here is a list of minimal tools you should install to build the Wing repo in your development
+environment:
+
+* [Node.js] version 18.x or above (we recommend [volta])
+* [Rust]
+* [AWS CLI] (only needed for integration tests - make sure to do the setup part to create
+  credentials)
+* [Terraform CLI] (only needed for integration tests)
+
+Installation:
+
+```sh
+$ git clone git@github.com:winglang/wing.git
+$ cd wing
+$ npm install
+```
+
+This is required once:
+
+```sh
+sudo ./scripts/setup_wasi.sh
+```
+
+[Nx]: https://nx.dev/
+[Node.js]: https://nodejs.org/en/
+[Rust]: https://www.rust-lang.org/tools/install
+[AWS CLI]: https://aws.amazon.com/cli/
+[Terraform CLI]: https://learn.hashicorp.com/terraform/getting-started/install.html
+[volta]: https://volta.sh
+
+## Development iteration
+
+The `nx wing` command can be executed from anywhere in the repository in order to build and run the
+compiler, SDK and the Wing CLI. Nx is configured to make sure only the changed components are built
+every time.
```

Comment:
nx runs the commands of the project you're currently it. It's expected that your current directory is the one you're currently concerned with.
```suggestion
The `nx wing` command can be executed from the root of the repository in order to build and run the
compiler, SDK and the Wing CLI. Nx is configured to make sure only the changed components are built
every time.
```

---

Alternatively:

```suggestion
The `nx wing winglang` command can be executed from anywhere in the repository in order to build and run the
compiler, SDK and the Wing CLI. Nx is configured to make sure only the changed components are built
every time.
```

nx [target] [project] can be run anywhere. Changing it back to `nx dev winglang` wouldn't look too bad.